styles/config/vocabularies/Technical/accept.txt

```diff
@@ -1,14 +1,17 @@
 [Aa]llowlist[s]?
 [Aa]utocommit
 [Aa]utodeploy
+[Aa]bstractive
 [Aa]ccessor[s]?
 ACL[s]?
+[Aa]udiobook[s]
```

Comment:
what's the trailing `[s]` for?

Reply:
For the plural.

Reply:
in regex, a bracket expression matches any one of the specific set of specified characters. for example. 'abc[xyz]' matches the strings "abcx", "abcy", and "abcz". it does not match "abc".

you can test your regexes on sites like [regex101](https://regex101.com/). we should also generally be testing new regex-based vale rules by writing a small test file with a bunch of correct and incorrect examples on different lines, and running vale against it.

Reply:
Had missed adding `?`. Tested this using [regex101](https://regex101.com/).